R/tags.R

```diff
@@ -382,17 +383,27 @@ tagWrite <- function(tag, textWriter, indent=0, eol = "\n") {
 
   # compute indent text
   indentText <- paste(rep(" ", indent*2), collapse="")
+  textWriter$writeWS(indentText)
```

Comment:
Was this an intentional change to always write the indent regardless of whether or not tag is a character?